docs/plugins.rst

```diff
@@ -0,0 +1,131 @@
+The BASIN-3D USGS plugin can be used to find USGS HUC information and identifiers:
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-14020001'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-21COL001-000058 --- TAYLOR RIVER AT ALMONT
+    EPA-21COL001-000078 --- EAST RIVER AT CONFL. WITH TAYLOR
+    EPA-21COL001-000150 --- SLATE RIVER ABOVE COAL CREEK
+    ...
+
+Option 2: Specify one or more EPA Site IDs, prefixed by "EPA-":
+
+.. code-block::
+
+    >>> monitoring_features = synthesizer.monitoring_features(parent_feature=['EPA-CORIVWCH_WQX-176', 'EPA-11NPSWRD_WQX-BLCA_09128000'])
+    >>> for monitoring_feature in monitoring_features:
+    >>>     print(f'{monitoring_feature.id} --- {monitoring_feature.name}')
+    EPA-11NPSWRD_WQX-BLCA_09128000 --- GUNNISON RIVER BELOW GUNNISON TUNNEL, CO
+    EPA-CORIVWCH_WQX-176 --- Uncompahgre -  Confluence Park
+
+
+5. Request time series data. Query argument aggregation_duration supports "NONE" or "DAY". See `epa_mapping.py <https://github.com/BASIN-3D/basin3d/blob/main/basin3d/plugins/epa_mapping.py>`_:
+
+.. code-block::
+
+    >>> measurement_timeseries_tvp_observations = synthesizer.measurement_timeseries_tvp_observations(monitoring_feature=['EPA-CCWC-COAL-26', 'EPA-CCWC-MM-29 WASH #3'], observed_property=['As', 'WT', 'DO'], start_date='2010-01-01', end_date='2011-01-01', aggregation_duration='NONE')
+
+6. Synthesized data should be cited following the Water Quality Portal data use policies.
+
+
+Section 3: Usage Notes
+^^^^^^^^^^^^^^^^^^^^^^
+.. warning::
+  **BASIN-3D capabilities that cannot be supported or are limited for the EPA WQX data source include:**
+
+    | - No unit conversions are performed for data values. Each :class:`basin3d.core.models.MeasurementTimeseriesTVPObservation` object will have consistent units for its data values. The EPA WQX units are reported in the :class:`basin3d.core.models.MeasurementTimeseriesTVPObservation` unit_of_measurement attribute and may not match the specified BASIN-3D vocabulary unit. Note: Unit conversions are performed for any depth / height information.
+    |
+    | - Timestamps are reported in both Standard and Daylight Savings time. When supplied, the utc_offset is reported in the :class:`basin3d.core.models.TimeValuePair` timestamp attribute following the ISO format. Because the utc_offset changes during the year, no value is reported in the :class:`basin3d.core.models.MeasurementTimeseriesTVPObservation` utc_offset attribute.
+    |
+    | - Only instantaneous and daily time aggregations are currently supported. Daily aggregations are supplied by the data source, not aggregated by BASIN-3D. All data with no "ResultTimeBasisText" reported are considered instantaneous.
```

Comment:
what does `"ResultTimeBasisText"` mean?  I can't find it anywhere else in our documentation.

Reply:
Clarified and referenced Section 4.